internal/thelma/clients/sherlock/state_writer.go

```diff
@@ -347,7 +356,7 @@ func toModelCreatableEnvironment(env terra.Environment, chartReleasesFromTemplat
 		RequiresSuitability:       utils.Nullable(env.RequireSuitable()),
 		TemplateEnvironment:       env.Template(),
 		HelmfileRef:               utils.Nullable(helmfileRef),
-		ChartReleasesFromTemplate: &chartReleasesFromTemplate,
+		AutoPopulateChartReleases: &autoPopulateChartReleases,
```

Comment:
Unrelated fallout from more recent Sherlock versions no longer allowing this field for backwards compatibility (we had this hardcoded to false at the only call-site, anyway)